grouper/graph.py

```diff
@@ -611,6 +615,13 @@ def get_permission_details(self, name, expose_aliases=True):
                         else:
                             data["service_accounts"][account] = {"permissions": [details]}
 
+            # Add permission audit value
+            permission_audited = False
+            for p in self._permissions.values():
+                if p.name == name:
+                    permission_audited = p.audited
+
+            data["audited"] = {"audited": permission_audited}
```

Comment:
this would mean `data["audited"]["audited"]` is a bool? can you provide some context as to what the thinking was here, as the double-nesting seems a bit strange.

Reply:
Agreed that it's a bit odd! I just wanted to keep the return values consistent with the function type signature.
# type: (str, bool) -> Dict[str, Dict[str, Any]]
I can simplify if this is confusing

Reply:
Given that this becomes API surface area we need to support "forever", externally logical (e.g. just make it a bool value, not a nested dict) beats keeping one type signature the same